### ISSUE TITLE
Add i18n in the rule 'order-in-components'

### ISSUE THREAD

**Issue by mickaelchanrion** [vuejs/eslint-plugin-vue#1231]
**Please describe what the rule should do:**
Maybe we could add the property 'i18n' in this [rule](https://eslint.vuejs.org/rules/order-in-components.html#vue-order-in-components) at the end of the list? 
I believe a lot of people are using vue-i18n so it could be nice to enforce a consistent position of that property. At the end because this property is related the rendering and can contain a lot of content.

**What category should the rule belong to?**

- [X] Enforces code style
- [ ] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
```vue
<script>
/* ✗ BAD */
export default {
  i18n: {
    messages: {
      // Lot of content here
      // making the scroll huge 
      // between the template 
      // and the logic of the component
    }
  },
  data() {
    return {};
  },
  methods: {}
};
</script>
```

```vue
<script>
/* ✓ GOOD */
export default {
  data() {
    return {};
  },
  methods: {},
  ii18n: {
    messages: {
      // Lot of content here
      // making the scroll huge 
      // between the template 
      // and the logic of the component
    }
  },
};
</script>
```

**Comment by ota-meshi** [comment]
Thank you for this suggestion.

I personally like to include `i18n` settings, but I think `vue-i18n` is a third party library, so I'm wondering if it should be included.

@kazupon What do you think?

**Comment by kazupon** [comment]
>I personally like to include i18n settings, but I think vue-i18n is a third party library, so I'm wondering if it should be included.

I think so too.
It should be 3rd party library.
vue-i18n has [eslint-plugin-vue-i18n](https://github.com/intlify/eslint-plugin-vue-i18n).
You can send PR, welcome contribution! :)

**Comment by ota-meshi** [comment]
@kazupon Thank you for your comment.

@mickaelchanrion I think it's good not to add the `i18n` option to the default settings of this plugin, so I'll close this issue.
Thank you for your understanding.
